decimal.md

```diff
@@ -14,7 +14,9 @@ In CockroachDB, the following are aliases for `DECIMAL`:
 
 ## Precision and Scale
 
-Precision (the maximum count of digits in a whole number, both to the left and right of the decimal point) and scale (the maximum count of digits to the right of the decimal point) are always unlimited for a `DECIMAL` value. If you specify these limits by declaring `DECIMAL(precision, scale)` on a column, they will not be enforced. 
+To limit a decimal column, use `DECIMAL(precision, scale)`, where `precision` is the maximum count of digits both to the left and right of the decimal point, and `scale` is the maximum count of digits to the right of the decimal point.
+
+When inserting a decimal value, if the digits to the right of the decimal point exceed the column's `scale`, CockroachDB rounds to the scale. After rounding, if the total count of digits exceeds the column's `precision`, CockroachDB gives an error.  
```

Comment:
The scale is the precise count of digits to the right of the decimal point. That is, `DECIMAL(5,4)` will always have exactly 4 digits to the right of the decimal point.

```
pmattis=# select 1::decimal(5,4);
 numeric
---------
  1.0000
```

We don't seem to handle this correctly yet.